src/adaptor.py

```diff
@@ -190,8 +207,8 @@ def handler(json_request, outgoing):
     # we have a valid request :)
     LOG.info("valid request")
 
-    params = subdict(request, ['action', 'id', 'token', 'version'])
-    params['force'] = request.get('force') # optional value
+    params = subdict(request, ['action', 'id', 'token', 'version', 'force', 'validate-only'])
+    params = renkeys(params, [('validate-only', 'dry_run')])
```

Comment:
those are function parameters and not part of the json response. here, the request is converted into something more suitable for internal use, renaming `validate-only`, to `dry_run`. in retrospect I should have just used 'dry-run' across the board. It evolved from being a discreet action like ingest or publish to being an option.